src/client/components/Footer.tsx

```diff
@@ -146,24 +146,40 @@ export const Footer = () => (
         <div css={ulWrapperStyles}>
           <ul css={[ulStyles, columnStyles]}>
             <li>
-              <Link cssOverrides={linkStyles} href={locations.PRIVACY}>
+              <Link
```

Comment:
What do you think about a `<ExternalLink>` component that adds `rel=noreferrer` 🤔  Would this make it more explicit that you were linking off site ? Could we also update the namespace in the routes, ie `locations.external.CONTACT_US` ?

Reply:
I like that idea! I'll make a separate PR for that suggestion.

Reply:
https://github.com/guardian/gateway/pull/1107